master/buildbot/config.py

```diff
@@ -121,7 +121,7 @@ def loadConfigDict(basedir, configFileName):
     sys.path.append(basedir)
     try:
         try:
-            exec(f, localDict)
```

Comment:
Problem of this is that the stacktrace will not generate proper stack traces, and will not tell the problem is in master.cfg.

This suggests to use compile(code, filename, 'exec') if PY3

http://stackoverflow.com/questions/436198/what-is-an-alternative-to-execfile-in-python-3